internal/exec/shell_utils.go

```diff
@@ -183,11 +184,20 @@ func execTerraformShellCommand(
 		if len(shellCommand) == 0 {
 			bashPath, err := exec.LookPath("bash")
 			if err != nil {
-				return err
+				// Try fallback to sh if bash is not available
+				shPath, shErr := exec.LookPath("sh")
+				if shErr != nil {
+					return fmt.Errorf("no suitable shell found: %v", shErr)
+				}
+				shellCommand = shPath
+			} else {
+				shellCommand = bashPath
 			}
-			shellCommand = bashPath
 		}
-		shellCommand = shellCommand + " -l"
+
+		if shellCommand == "/bin/zsh" {
```

Comment:
I think we need to generalize this. The shell path can vary by OS. At a minimum we should use basename to look at what shell. @coderabbitai any tips?